extensions/ql-vscode/src/databases.ts

```diff
@@ -858,12 +858,12 @@ export class DatabaseManager extends DisposableObject {
   }
 
   private updatePersistedCurrentDatabaseItem(): void {
-    this.ctx.workspaceState.update(CURRENT_DB, this._currentDatabaseItem ?
+    void this.ctx.workspaceState.update(CURRENT_DB, this._currentDatabaseItem ?
       this._currentDatabaseItem.databaseUri.toString(true) : undefined);
   }
 
   private updatePersistedDatabaseList(): void {
-    this.ctx.workspaceState.update(DB_LIST, this._databaseItems.map(item => item.getPersistedState()));
+    void this.ctx.workspaceState.update(DB_LIST, this._databaseItems.map(item => item.getPersistedState()));
```

Comment:
Perhaps this and `updatePersistedCurrentDatabaseItem` above should be `async` and these should `await` so that callers of them can have the option of waiting for the update to complete if they want to. I guess it doesn't matter if none of the callers _need_ to wait for it, but it kind of seems like this is a case where we _might_ want to wait and should make it clear whether we're waiting or not.

Reply:
Actually, looking at the calling code, we probably want to await all of these calls. Made that change.